spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'User index', type: :feature do
+  before :each do
+    @user = User.create(name: 'olla', photo: 'https://picsum.photos/id/23/200', bio: 'web developer',
+                        posts_counter: 0)
+    visit root_path
+  end
+
+  it 'shows the correct content' do
+    expect(page).to have_content('Welcome To User Page')
+  end
+
+  it 'shows the number of posts each user has written' do
+    expect(page).to have_content('Number of posts')
+  end
+
+  it 'When I click on a user it should redirect to user/show page' do
+    click_on 'olla'
+    expect(page).to have_content 'olla'
+  end
```

Comment:
- [ ] Please test that you can see the **actual number of posts each user has written** not for the presence of the words "Number of posts" like you have done above 🙏 

- [ ] [OPTIONAL] Well done for checking for content on the user's show page. However, I want to suggest that a more fail-proof test would be to check the current path as you can see below 👇 
```ruby
# Expect to be redirected to the user's show page
expect(current_path).to eq(user_path(user))
```